### DISCUSSION TITLE
Oracle Integration Request

### DISCUSSION THREAD

**Discussion by vishal-reddy** [PrefectHQ/prefect#4395] [Ideas]
I did not see an oracle integration available. How difficult would this be to develop?

**Comment by zanieb** [comment]
Hey @vishal-reddy -- you'll have to explain in a bit more detail what you're looking for. Oracle has a lot of products.

**Reply by vishal-reddy** [reply]
Oracle database. I see there are connectors such as postgres and mysql.

**Comment by mr-tester-william-lo** [comment]
very easy to implement,
I think you can write your own class for data select query and for insert statement
Example as below:
https://www.oracletutorial.com/python-oracle/inserting-data/

**Reply by mr-tester-william-lo** [reply]
Also, the reason i suggest you implement some of the connector yourself, is because what prefectIO developed doesn't work on any data integrity checking, which you may need to include in your prefect workflow
